projects/js-toolkit/packages/js-toolkit-scripts/src/scripts/create-jar/xml.ts

```diff
@@ -22,27 +22,27 @@ const TYPES = {
 export function addMetatypeAttr(
 	metatype: XmlObject,
 	id: string,
-	desc: ConfigurationJsonField
+	description: ConfigurationJsonField
```

Comment:
Since this is of type `ConfigurationJsonField` I would call it `field` of `fieldDescription` better...

It's ironic that I was the one naming it `desc` and now I request a more proper name :muscle: :joy: 

However it has an explanation: this was ported from JS to TS and, in TS, there was no `ConfigurationJsonField` type :-)